portfolio/src/main/webapp/index.html

```diff
@@ -39,10 +39,15 @@ <h2>Random Facts about me</h2>
         <textarea name="comment-box" rows="6" cols="100"></textarea>
         <button type="submit">Submit comment</button>   
       </form> 
-      <p>Comments:</p>
+      <form class="space-between-elements" onsubmit="getCommentThread(); return false;">
```

Comment:
This works, but it's not that great. You're submitting the form and then overriding the submit behavior to call getCommentThread() and then stop the submit by returning false. All of your behavior is in the method, so you don't need the form submit at all. You could just leave out the `form` altogether and just replace your submit button with `<button type="button" onclick="getCommentThread()">..` which will just call the method directly without all of the form submit behavior.

Basically, what you did that for the delete button